dataflux_pytorch/benchmark/checkpointing/multinode/train.py

```diff
@@ -97,14 +130,24 @@ def main(ckpt_dir_path: str, ckpt_restore_path: str = ""):
     avg_save_time = (end - start) / num_save_calls
     num_load_calls = int(os.environ.get("NUM_LOAD_CALLS", 3))
     load_checkpoint_times = []
+    if args.save_only:
+        print("Skipping loads because you set --save_only")
+        num_load_calls = 0
+        load_checkpoint_times = [0]
+    elif args.load_only:
+        print(f"Copying contents of {ckpt_dir_path} to {ckpt_restore_path}")
+        copy_bucket_to_local(ckpt_dir_path.removeprefix("gs://"),
```

Comment:
It may not matter, but you already should have a storage client configured within the model at this point right? Not sure if it matters whether or not we open two though.